docs/getting-started/09-trigger-microservice-with-event.md

```diff
@@ -72,19 +69,19 @@ Console UI
 
 2. Go to **Discovery and Network** > **Services** in the left navigation panel and select `orders-service`.
 
-3. Once in the service's details view, select **Add Event Trigger** in the **Event Triggers** section.
+3. Once in the Function's details view, switch to the **Configuration** tab and select **Create Event Subscriptions** in the **Event Subscriptions** section.
```

Comment:
Actually, that doc is about the service, not a Function.

Reply:
You're right, it was fixed in e32f721